README.md

```diff
@@ -27,14 +46,33 @@ make docs
 ```
 The documentation can then be found at `docs/html/index.html`
 
+## Tests
+To run unit tests and jlc C compilation tests, execute
+```
+make check
+```
+
+The tests can also be run instrumented under valgrind to validate absence
+of detectable memory errors:
+```
+make valgrind-check
+```
+
+Lastly, when the build has been configured with coverage support, then
+the following build target will compute unit test coverage for all files:
```

Comment:
Could you be more precise here, please? I just stumbled over the problem that I did not have it build with coverage enabled and invoked `make coverage`. The output `No rule to make target coverage` is not very helpful. Could you clarify that one would need to configure (and build) with `--enable-coverage`, please?

Reply:
documented the flag